table/server/common/src/test/java/alluxio/table/common/udb/PathTranslatorTest.java

```diff
@@ -76,24 +76,24 @@ public void multipleSubdir() throws Exception {
 
   @Test
   public void samePathDifferentUfs() throws Exception {
-    String alluxioPath  = "alluxio:///my/table/directory";
+    String alluxioPath  = "alluxio://master/my/table/directory";
     String ufsPath = "/the/ufs/location";
-    mTranslator.addMapping(alluxioPath, "ufs_a://" + ufsPath);
+    mTranslator.addMapping(alluxioPath, "ufs-a://" + ufsPath);
```

Comment:
why was this changed? does '_' not work?

Reply:
`java.lang.IllegalArgumentException: java.net.URISyntaxException: Illegal character in scheme name at index 3: ufs_a:/the/ufs/location`

It seems "_" is forbidden in schemes.

Reply:
yup, it makes sense now that I see you create AlluxioURI out of the path